mlos_bench/mlos_bench/tests/config/schemas/tunable-params/test_tunable_params_schemas.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Tests for tunable params schema validation.
+"""
+
+from os import path
+from typing import Dict
+
+import jsonschema
+import pytest
+
+from mlos_bench.config.schemas import ConfigSchema
+
+from mlos_bench.tests.config.schemas import get_schema_test_cases, SchemaTestCaseInfo
+
+
+# General testing strategy:
+# - hand code a set of good/bad configs (useful to test editor schema checking)
+# - for each config, load and validate against expected schema
+
+TEST_CASES: Dict[str, SchemaTestCaseInfo] = get_schema_test_cases(path.join(path.dirname(__file__), "test-cases"))
```

Comment:
maybe make these variables pytest fixtures?
```suggestion
@pytest.fixture
def list_test_cases() -> Dict[str, SchemaTestCaseInfo]:
    return get_schema_test_cases(path.join(path.dirname(__file__), "test-cases"))
```
etc.

Reply:
I don't think that works with `__file__` since then `__file__` is the `conftest.py` instead of the `test_FOO_schemas.py`, but I agree, there's a bunch of boilerplate repeat code between these different schema tests that could probably be reduced in some way.
That was part of why I structured it this way though - I wanted them to more or less follow the same pattern.  It's changed a little since I first started playing with it though.  Anyways, lemme play somemore and see what I can come up with.

Reply:
Did some reorging to reduce the repetition.  Lemme know what you think.  Thanks!

Reply:
Gonna call this good for now.  We can address it more in subsequent schema PRs if you want.